pkg/network/cache/infocache.go

```diff
@@ -114,6 +126,27 @@ func newPodInterfaceCacheStore(vmi *v1.VirtualMachineInstance, baseDir, pattern
 	return podInterfaceCacheStore{vmi: vmi, baseDir: baseDir, pattern: pattern}
 }
 
+type vmNetworkingInfraInterfacesStore struct {
+	pid     string
+	pattern string
+	baseDir string
+}
+
+func (d vmNetworkingInfraInterfacesStore) Read(iface string) (*networkdriver.VIF, error) {
+	cachedIface := &networkdriver.VIF{}
+	err := readFromCachedFile(cachedIface, getInterfaceCacheFile(d.baseDir, d.pattern, d.pid, iface))
+	return cachedIface, err
+}
+
+func (d vmNetworkingInfraInterfacesStore) Write(ifaceName string, ifaceToCache *networkdriver.VIF) (err error) {
+	err = writeToCachedFile(ifaceToCache, getInterfaceCacheFile(d.baseDir, d.pattern, d.pid, ifaceName))
+	return
+}
```

Comment:
Is it worth to add a little helper with

```golang
func (d vmNetworkingInfraInterfacesStore) getInterfaceCacheFile(ifaceName string) string {
    return getInterfaceCacheFile(d.baseDir, d.pattern, d.pid, ifaceName)
}
```

So we will do
```golang
func (d vmNetworkingInfraInterfacesStore) Read(ifaceName string) (*networkdriver.VIF, error) {
	cachedIface := &networkdriver.VIF{}
	err := readFromCachedFile(cachedIface, d.getInterfaceCacheFile(ifaceName))
	return cachedIface, err
}

func (d vmNetworkingInfraInterfacesStore) Write(ifaceName string, ifaceToCache *networkdriver.VIF) (err error) {
	return writeToCachedFile(ifaceToCache, d.getInterfaceCacheFile(ifaceName))
}
```

Reply:
Done.